docs/redis-clients/python.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Python guide"
+linkTitle: "Python"
+description: Connect your Python application to a Redis database
+weight: 1
+
+---
+
+Use one of the following options to connect your application to a Redis database:
+
+* [redis-py](#redis-py)
+* [redis-om-python](#redis-om-python)
+
+## redis-py
+
+Get started with a Python client for Redis. Looking for a high-level library to handle object mapping? See [redis-om-python](#redis-om-python).
+
+`redis-py` requires a running Redis server. See the [Getting started](/docs/getting-started/) for Redis installation instructions.
+
+### Install
+
+To install `redis-py`, type:
+
+```bash
+pip install redis
+```
+
+For faster performance, install redis with [`hiredis`](https://github.com/redis/hiredis) support. This provides a compiled response parser, and for most cases requires zero code changes. By default, if `hiredis` >= 1.0 is available, `redis-py` attempts to use it for response parsing.
+
+```bash
+pip install redis[hiredis]
+```
+
+### Connect to your Redis database
+
+By default, `redis-py` uses a connection pool to manage connections. Each instance of a Redis class receives its own connection pool. You can, however, define your own `redis.ConnectionPool`.
+
+```sh
+>>> pool = redis.ConnectionPool(host='localhost', port=6379, db=0)
+>>> r = redis.Redis(connection_pool=pool)
+```
+
+Alternatively, you might want to look at [async connections](https://redis.readthedocs.io/en/stable/examples/asyncio_examples.html), or [cluster connections](https://redis.readthedocs.io/en/stable/connections.html#cluster-client), or [async cluster connections](https://redis.readthedocs.io/en/stable/connections.html#async-cluster-client).
```

Comment:
Async code is gaining more popularity, so it probably makes sense to add a second example with async code and reference client docs for more detailed/advanced examples afterward.

Reply:
[](url)